src/components/calendar.rs

```diff
@@ -21,6 +23,8 @@ pub struct CalendarMonthProperties {
     pub weekday_start: Weekday,
 }
 
+static CURRENT_LOCALE_CELL: OnceLock<Locale> = OnceLock::new();
```

Comment:
Currently the first stable rust version that supports "OnceLock" is 1.70.0 which is the same version listed in the Cargo.toml file of patternfly_yew.
However, just to be on the safe side, I tried compiling patternfly_yew with version 1.70.0 and discovered that it doesn't compile because a dependency requires 1.73.0.

```
patternfly-yew> cargo +1.70.0 build
error: package `bumpalo v3.16.0` cannot be built because it requires rustc 1.73.0 or newer, while the currently active rustc version is 1.70.0


│       └── windows_x86_64_msvc v0.52.6
├── gloo-events v0.2.0
│   ├── wasm-bindgen v0.2.93
│   │   ├── cfg-if v1.0.0
│   │   ├── once_cell v1.19.0
│   │   └── wasm-bindgen-macro v0.2.93 (proc-macro)
│   │       ├── quote v1.0.37
│   │       │   └── proc-macro2 v1.0.86
│   │       │       └── unicode-ident v1.0.12
│   │       └── wasm-bindgen-macro-support v0.2.93
│   │           ├── proc-macro2 v1.0.86 (*)
│   │           ├── quote v1.0.37 (*)
│   │           ├── syn v2.0.76
│   │           │   ├── proc-macro2 v1.0.86 (*)
│   │           │   ├── quote v1.0.37 (*)
│   │           │   └── unicode-ident v1.0.12
│   │           ├── wasm-bindgen-backend v0.2.93
│   │           │   ├── bumpalo v3.16.0                      <---------
│   │           │   ├── log v0.4.22
│   │           │   ├── once_cell v1.19.0
│   │           │   ├── proc-macro2 v1.0.86 (*)
│   │           │   ├── quote v1.0.37 (*)
│   │           │   ├── syn v2.0.76 (*)
│   │           │   └── wasm-bindgen-shared v0.2.93
│   │           └── wasm-bindgen-shared v0.2.93
│   └── web-sys v0.3.70
│       ├── js-sys v0.3.70

```

Reply:
So it needs to be upticked, but not due to this PR :) Would be great if you could do it anyway. But ok if not.

Reply:
Well then I will change the rust-version of the Cargo.toml file of patternfly yew to 1.73.